shared_requirements.txt

```diff
@@ -185,6 +185,7 @@ opentelemetry-sdk<2.0.0,>=1.5.0,!=1.10a0
 #override azure-eventhub-checkpointstoretable azure-core<2.0.0,>=1.14.0
 #override azure-eventhub uamqp>=1.6.0,<2.0.0
 #override azure-appconfiguration azure-core<2.0.0,>=1.24.0
+#override azure-appconfiguration isodate
```

Comment:
I see you don't specify other version ranges than the general one, do we need to override then?